content/en/docs/faq.md

```diff
@@ -142,3 +142,68 @@ in the result, you’ll receive the tenant-kubeconfig file, which you can provid
 Here you can find reference repository to learn how to configure Cozystack services using GitOps approach:
 
 - https://github.com/aenix-io/cozystack-gitops-example
+
+### How to rotate CA
```

Comment:
In what cases do we need to rotate the CA keys? How do we determine if we need to do it now?

What is the default validity period of a CA key?